site/docs/latest/index.md

```diff
@@ -0,0 +1,12 @@
+---
+title: Apache Pulsar Documentation
+layout: docs-main
+---
+
+Welcome to the documentation for [Pulsar]({{ site.baseurl }}), an open-source distributed pub-sub messaging system originally created by [Yahoo](http://yahoo.github.io) and now under the stewardship of the [Apache Software Foundation](http://www.apache.org/).
+
+## Translations
+
+The Pulsar documentation is available in the following languages:
+
+<img src="{{ site.baseurl }}img/japan-flag.svg" class="mt-0 mb-0" style="border: 1px solid #ddd; width: 4rem; margin-right: 1.25rem;"> [Japanese]({{ site.baseurl }}ja)
```

Comment:
I think it gets generated correctly, `site.baseUrl` gets expanded at generation time.